client/src/GlobalStyles.js

```diff
@@ -0,0 +1,43 @@
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyle = createGlobalStyle`
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap');
+
+  text-align: center;
+  font-weight: 400;
+
+:root {
+    --primary: #F8F8FF;
+    --secondary: #515154;
+    --tertiary: #CBCBD3;
+    --bitcoinColor: #F7931A;
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  color: #1D1D1F;
+}
+
+body * {
+  font-family: 'Roboto', sans-serif;
+  color: #1D1D1F;
+  font-size: 14px;
+}
```

Comment:
What was your intention with this? 
With the `body * ` selector you select *every* child of the body and provide it with this styling. 
I would recommend to not use this. Font related stylings are always inherited by the children, so it is sufficient to define these styles in the body selector in line 25.

Reply:
Changed, thank you!